src/ui/public/notify/notifier.js

```diff
@@ -271,7 +271,7 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  */
 Notifier.prototype.error = function (err, cb) {
-  return add({
+  return this.add({
```

Comment:
I think this is going to cause issues with several instances of `do().catch(notify.fatal)` in Kibana. Maybe we should just keep a sharable reference to add for internal use, but also export it on the prototype?